docs/assistant/security-assistant.asciidoc

```diff
@@ -2,4 +2,97 @@
+
+[discrete]
+[[start-chatting]]
+== Start chatting
+
+To open the Security Assistant, press *Cmd + ;* (or *Ctrl + ;* in Windows) from anywhere in the {security-app}. This opens the *Welcome* conversation context, where you can ask general questions about {elastic-sec}.
+
+You can also chat with the Security Assistant from several areas of {elastic-sec}, and context-specific data and prompts will populate your conversation.
+
+* <<view-alert-details, Alert details>> or Event details flyout: Click *Chat* while viewing the details of an alert or event.
+* <<rules-ui-management, Rules page>>: Select one or more rules, then click the magic wand icon (🪄✨) at the top of the page next to the *Rules* title.
+* <<data-quality-dash, Data Quality dashboard>>: Select the *Incompatible fields* tab, then click *Chat*. (This is only available for fields marked red, indicating they're incompatible).
+* <<timelines-ui, Timeline>>: Select the *Security Assistant* tab.
+
+All chat history persists within your session, so you can navigate away to other areas of {elastic-sec}, then later return to ongoing conversations.
+
+[discrete]
+[[interact-with-assistant]]
+== Interact with the Security Assistant
+
+Use these features to adjust and act on your conversations with the Security Assistant:
+
+* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the Security Assistant's answers to be.
++
+[role="screenshot"]
+image::images/system-prompt.gif[The system prompt drop-down menu,90%]
++
+NOTE: The system prompt is only configurable at the start of a conversation. To reconfigure, clear the chat and start a new conversation.
+
+* Select a _quick prompt_ at the bottom of the chat window to get help writing a prompt for a specific purpose, such as summarizing an alert or converting a query from a legacy SIEM to {elastic-sec}. Available quick prompts vary based on context. You can also add your own custom quick prompts for questions that you frequently ask the Security Assistant.
++
+[role="screenshot"]
+image::images/quick-prompts.png[Quick prompts highlighted below a conversation,90%]
+
+* Use available buttons to perform actions in the conversation history and prompt entry area:
+
+** *Add note to timeline* (image:images/icon-add-note.png[Add note icon,16,16]): Create a note in Timeline using the selected text.
+** *Add to existing case* (image:images/icon-add-to-case.png[Add to case icon,19,16]): Add a comment to an existing case using the selected text.
+** *Copy to clipboard* (image:images/icon-copy.png[Copy to clipboard icon,17,18]): Copy the text to clipboard to paste elsewhere. This is also helpful for resubmitting a previous prompt.
+** *Add to timeline* (image:images/icon-add-to-timeline.png[Copy to clipboard icon,17,18]): Add a filter or query to Timeline using the text. This button appears for certain queries in the Security Assistant's responses.
```

Comment:
I think I mentioned this in chat, but this action can be finicky since a specific `query language keyword` must be present in the text before the codeblock for it to correctly determine the language and function (if the query itself was even valid). Perhaps too much for the docs, but would be useful in a `tips` section. Thankfully you can work with the assistant by adjusting the prompt to have it mention the specific query language keyword before the codeblock. The keywords for EQL, KQL and DSL are [as follows](https://github.com/elastic/kibana/blob/95951b95956132e95bb6baf7618b8943618e6f89/x-pack/packages/kbn-elastic-assistant/impl/assistant/use_conversation/helpers.ts#L34-L38):

```
  const types = {
    eql: ['Event Query Language', 'EQL sequence query'],
    kql: ['Kibana Query Language', 'KQL Query'],
    dsl: ['Elasticsearch QueryDSL', 'Elasticsearch Query DSL', 'Elasticsearch DSL'],
  };
```
e.g.:

```
U: Can you generate an `Event Query Language` query that looks for four failed logons followed by a successful logon?
U: Can you generate an Elasticsearch DSL filter for host.name: "REALLY BAD HOST"? (Be sure to leave off the "query" object wrapper since we just want the filter)
```

Reply:
Hm, good to know. It might still be worth mentioning even in the initial docs, to help ensure that this feature works as expected for previewing users.
```suggestion
** *Add to timeline* (image:images/icon-add-to-timeline.png[Copy to clipboard icon,17,18]): Add a filter or query to Timeline using the text. This button appears for certain queries in the Security Assistant's responses. 
+
TIP: Be sure to specify which query language you'd like the Security Assistant to use for queries. For example: "Can you generate an Event Query Language query to find four failed logins followed by a successful login?"
```